src/pcapi/notifications/push/user_attributes_updates.py

```diff
@@ -21,37 +28,53 @@ def get_user_attributes(user: User) -> dict:
     return {
         "u.credit": int(credit.all.remaining * 100) if credit else 0,
         "u.departement_code": user.departementCode,
-        "date(u.date_of_birth)": user.dateOfBirth.strftime(BATCH_DATETIME_FORMAT) if user.dateOfBirth else None,
+        "date(u.date_of_birth)": _format_date(user.dateOfBirth),
         "u.postal_code": user.postalCode,
-        "date(u.date_created)": user.dateCreated.strftime(BATCH_DATETIME_FORMAT),
+        "date(u.date_created)": _format_date(user.dateCreated),
         "u.marketing_push_subscription": user.get_notification_subscriptions().marketing_push,
         "u.is_beneficiary": user.isBeneficiary,
-        "date(u.deposit_expiration_date)": user.deposit_expiration_date.strftime(BATCH_DATETIME_FORMAT)
-        if user.deposit_expiration_date
-        else None,
+        "date(u.deposit_expiration_date)": _format_date(user.deposit_expiration_date),
     }
 
 
-def format_booking_date(booking_date: datetime) -> Optional[str]:
-    return booking_date.strftime(BATCH_DATETIME_FORMAT) if booking_date else None
-
-
 def get_user_booking_attributes(user: User) -> dict:
     from pcapi.core.users.api import get_domains_credit
-    from pcapi.core.users.api import get_last_booking_date
-    from pcapi.core.users.repository import get_booking_categories
 
-    credit = get_domains_credit(user)
-    last_booking_date = get_last_booking_date(user)
-    booking_categories = get_booking_categories(user)
+    user_bookings = (
+        Booking.query.options(
+            joinedload(Booking.stock)
+            .joinedload(Stock.offer)
+            .load_only(Offer.type, Offer.url)
+            .joinedload(Offer.product)
```

Comment:
Si je comprends bien, on fait ça pour accéder à `booking.stock.offer.product.id` plus bas dans cette fonction. Mais si on accédait à `...offer.productId` à la place, on pourrait ici éviter le `joinedload(Offer.product)` :

```suggestion
            .load_only(Offer.type, Offer.url, Offer.productId)
```